server/src/main/java/org/elasticsearch/cluster/coordination/Coordinator.java

```diff
@@ -1398,6 +1397,13 @@ public void onFailure(Exception e) {
             }, EsExecutors.newDirectExecutorService(), transportService.getThreadPool().getThreadContext());
         }
 
+        void cancelTimeoutHandlers() {
```

Comment:
NIT: can be `private`?

Reply:
++ b7e1d8a. I haven't been able to work out how to get IntelliJ to tell me that even when making it private means the compiler generates a package-private wrapper as it does here?